nextcloud_update.sh

```diff
@@ -552,6 +567,31 @@ fi
 # Recover apps that exists in the backed up apps folder
 run_script STATIC recover_apps
 
+# Restore app status
+if [ "${APPSTORAGE[0]}" != "no-export-done" ]
+then
+    print_text_in_color "$ICyan" "Restoring the status of apps. This can take a while..."
+    for app in "${!APPSTORAGE[@]}"
+    do 
+        if [ -n "${APPSTORAGE[$app]}" ]
+        then
+            if [ "${APPSTORAGE[$app]}" = "yes" ]
+            then
+                occ_command_no_check app:enable "$app"
```

Comment:
Why do you enable or disbale apps? The check should only match already enabled apps, right?

Reply:
The result for me is:

```
root@nextcloud500gb:~# bash test.sh 
Getting and backing up the status of apps for later. This can take a while...
import...
Restoring the status of apps. This can take a while...
nextcloud_announcements already enabled
firstrunwizard already enabled
settings already enabled
theming already enabled
activity already enabled
contactsinteraction already enabled
files_videoplayer already enabled
sharebymail already enabled
photos already enabled
comments already enabled
support already enabled
password_policy already enabled
systemtags already enabled
provisioning_api already enabled
federation already enabled
files_pdfviewer already enabled
extract already enabled
dav already enabled
files_sharing already enabled
text already enabled
serverinfo already enabled
mail already enabled
workflowengine already enabled
files_trashbin already enabled
calendar already enabled
files already enabled
files_versions already enabled
lookup_server_connector already enabled
files_rightclick already enabled
accessibility already enabled
oauth2 already enabled
twofactor_backupcodes already enabled
admin_audit already enabled
logreader already enabled
deck already enabled
No such app enabled: contacts
survey_client already enabled
federatedfilesharing already enabled
groupfolders already enabled
updatenotification already enabled
viewer already enabled
recommendations already enabled
privacy already enabled
notifications already enabled
cloud_federation_api already enabled
root@nextcloud500gb:~# 
```

Doesn't look good, imho.